rswag-specs/spec/rswag/specs/request_factory_spec.rb

```diff
@@ -103,6 +103,64 @@ module Specs
           end
         end
 
+        context "'query' parameters of type 'object'" do
+          let(:things) { {'foo': 'bar'} }
+
+          before do
+            metadata[:operation][:parameters] = [
+              {
+                name: 'things', in: :query,
+                style: style,
+                explode: explode,
+                schema: { type: :object, additionalProperties: { type: :string } }
+              }
+            ]
+            allow(example).to receive(:things).and_return(things)
```

Comment:
These are all OpenAPI 3 tests, so we should set the doc_version:

```suggestion
            allow(example).to receive(:things).and_return(things)
            let(:swagger_doc) { { swagger: '3.0' } }
```